lasagne/layers/pool.py

```diff
@@ -221,10 +227,22 @@ def get_output_for(self, input, **kwargs):
                                         st=self.stride,
                                         ignore_border=self.ignore_border,
                                         padding=self.pad,
+                                        mode=self.mode,
                                         )
         return pooled
 
 
+class MaxPool2DLayer(Pool2DLayer):  # for consistency
```

Comment:
This needs a docstring. Either copy what we had before, or tell that it's a shortcut for `Pool2DLayer` with `mode=max`. If possible, have a look at the documentation generated by Sphinx to check whether your solution is good.